server/controllers/getProducts.js

```diff
@@ -0,0 +1,8 @@
+const getProductQuery = require('../database/queries/products');
```

Comment:
you export the query function in inde.js as an object
```js
module.exports = {
    getProductsQuery
};
```
So you have to import here an object
```js
const {getProductQuery} = require('../database/queries/products');
```

Reply:
Good point @umarsalah, totally agree with you.